ember-cli-build.js

```diff
@@ -6,7 +6,12 @@ module.exports = function(defaults) {
     var app = new EmberApp(defaults, {
         'ember-bootstrap': {
             importBootstrapFont: false
-        }
+        },
+
+        babel: {
+            optional: ['es6.spec.symbols'],
```

Comment:
### Before
<pre>
File sizes:
 - flag-icon-406d9b826bcb858a49b38b51fe43b066.css: 36.81 KB (2.79 KB gzipped)
 - flag-icon.min-30b7e73df6634bc075fea5edd5191583.css: 32.7 KB (2.71 KB gzipped)
 - isp-286faafa0c151e1be5887c9a272d690f.css: 7.36 KB (1.68 KB gzipped)
 - isp-9c7949c09061ca6e6a21b45dd3a0c65c.js: 492.72 KB (105.87 KB gzipped)
 - vendor-875950d5ba29605078f992509451358c.js: 1.51 MB (338.9 KB gzipped)
 - vendor-ef22d01385b15d933d500890c9b115f3.css: 153.65 KB (28.35 KB gzipped)
</pre>

### After
<pre>
 - flag-icon-406d9b826bcb858a49b38b51fe43b066.css: 36.81 KB (2.79 KB gzipped)
 - flag-icon.min-30b7e73df6634bc075fea5edd5191583.css: 32.7 KB (2.71 KB gzipped)
 - isp-286faafa0c151e1be5887c9a272d690f.css: 7.36 KB (1.68 KB gzipped)
 - isp-7ab931d917e40f8dba6f638cd00ba3a9.js: 493.45 KB (106.19 KB gzipped)
 - vendor-7e50c356e3e5e1536d51d1de7da1a17e.js: 1.56 MB (357.7 KB gzipped)
 - vendor-ef22d01385b15d933d500890c9b115f3.css: 153.65 KB (28.35 KB gzipped)
</pre>

Total size of the dist directory before and after is 12M--the biggest difference being vendor.js with 18.8k added.